### PR TITLE
Allow to check checkable support

### PR DESCRIPTION
**Pull request by Montel**

### CODE REVIEW

**Review by tronical — APPROVED** [review]
Thanks Laurent

File: tests/cases/widgets/button.slint [hunk 1]
```diff
@@ -82,6 +83,7 @@ assert_eq!(bbb.accessible_label().unwrap(), "Bbb");
 assert_eq!(bbb.accessible_description().unwrap(), "Normal Button");
 assert_eq!(bbb.accessible_value(), None);
 assert_eq!(bbb.accessible_checked(), Some(false));
+assert_eq!(aaa.accessible_checkable(), Some(false));
```
Comment (ogoffart) [inline tests/cases/widgets/button.slint:86]:
```suggestion
assert_eq!(bbb.accessible_checkable(), Some(false));
```

File: tests/cases/widgets/button.slint [hunk 2]
```diff
@@ -128,6 +131,7 @@ auto bbb = label_search.front();
 assert_eq(bbb.accessible_label().value(), "Bbb");
 assert_eq(bbb.accessible_description().value(), "Normal Button");
 assert_eq(bbb.accessible_value(), std::nullopt);
+assert_eq!(aaa.accessible_checkable().value(), false);
```
Comment (ogoffart) [inline tests/cases/widgets/button.slint:134]:
```suggestion
assert_eq!(bbb.accessible_checkable().value(), false);
```

Comment (Montel) [reply tests/cases/widgets/button.slint:134]: oops.

**Comment by ogoffart** [comment]
Thanks

**Pull request merged**